meshapp/src/profile/profile-experience.tsx

```diff
@@ -0,0 +1,302 @@
+import { useState } from "react";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Typography,
+  createTheme,
+  ThemeProvider,
+  Grid,
+} from "@mui/material";
+
+import EditIcon from "@mui/icons-material/Edit";
+import AddIcon from "@mui/icons-material/Add";
+import SaveIcon from "@mui/icons-material/Save";
+import DeleteIcon from "@mui/icons-material/Delete";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+
+import ProfileTextField from "./profile-textfield";
+
+interface AccordionProps {
+  title: string;
+  description: string;
+  organization: string;
+  index: number;
+}
+
+const theme = createTheme({
+  palette: {
+    mode: "light",
+    primary: {
+      main: "#0b7d66",
+    },
+  },
+  typography: {
+    h1: {
+      fontFamily: "cocogoose",
+      fontWeight: "bold",
+      color: "#26383A",
+    },
+  },
+});
+
+/**
+ * Displays the user's experience.
+ *
+ * @param props - Properties of the component
+ * @param {string} props.accountID - accountID associated with the profile
+ */
+export default function ProfileExperience(props: { accountID: number }) {
+  const [editMode, setEditMode] = useState(false);
+  const [expandMode, setExpandMode] = useState([false, false]);
+  const [occupations, setOccupations] = useState([
+    "Animal Petter",
+    "Object Oriented Snob",
+  ]);
+  const [organizations, setOrgranizations] = useState([
+    "Self-Employed",
+    "MindGeek, Montreal Canada",
+  ]);
+  const [descriptions, setDescriptions] = useState([
+    "Pet cute animals",
+    "Lebron James",
+  ]);
+
+  const handleEditClick = () => {
+    setEditMode(true);
+  };
+
+  const handleSaveClick = () => {
+    setEditMode(false);
+    setOccupations([...occupations]);
+    setOrgranizations([...organizations]);
+    setDescriptions([...descriptions]);
+  };
+
+  const handleExpandClick = (index: number) => {
+    expandMode[index] = !expandMode[index];
+    setExpandMode([...expandMode]);
+    console.log(expandMode[index]);
+  };
+
+  const handleAddClick = () => {
+    setOccupations([...occupations, ""]);
+    setOrgranizations([...organizations, ""]);
+    setDescriptions([...descriptions, ""]);
+    setExpandMode([...expandMode, false]);
+  };
+
+  //TODO: Remove experience 
+  const handleDeleteClick = () => {
+
+  };
+
+  //TODO: Receive and store experience
+  function saveOccupation(text: string, index: number) {
```

Comment:
We should probably have some sort of input validation for these save-methods, or maybe for the corresponding `ProfileTextField`s